### ISSUE TITLE
Friction Cones Rotation bug

### ISSUE THREAD

**Issue by EnricoMingo** [robotology/OpenSoT#96]
@alaurenzi tried the Friction Cones constraints pointed out that the constraints wrongly rotates with the link which is in contact. Indeed in line 66 we get the actual pose of the link and in line 79 we multiply the updated constraint with the updated rotation.
A quick fix could be to just do it at the beginning when the link name is passed and eventually give the possibility to update the rotation by the user.
At the moment, the constraints works if the link remain always in conctact but in robots like CENTAURO the rotation needs to be removed.

**Comment by alaurenzi** [comment]
To me a good solution would be to provide an api to set the normal to the contact plane

**Comment by EnricoMingo** [comment]
I agree, then we can change the ```friction_cone``` pair as a ```Eigen::Vector3d``` and a ```double``` which are the normal and the ```mu``` value. We can then give a method to access the ```friciton_cones``` vector and change that values.

**Comment by EnricoMingo** [comment]
With the last commit in ```devel```, the orientation is set just in the beginning and not anymore updated.
